api/journaling_api.md

```diff
@@ -303,13 +303,20 @@ Link: </events/organizations/23294/integrations/54108/f89067f2-0d50-4bb2-bf78-20
 
 ### Limiting the size of the batch
 
-Depending on the frequency of the events in your registration, the number of events returned in a single response batch varies. A batch of events will always have one event but there is no upper limit to the number of events that can be returned in a single batch. In case you wish to set an upper bound, you can supply the `limit` query parameter with the maximum number of events that may be returned by the API.
+When events are created at a high frequency, Journal persists groups of events in its storages; 
+when events are created at a lower rate, this Journal persistent storages will contain only one event. 
+
+Hence, depending on the traffic of the events associated with your registration, 
+the number of events returned in a single response batch varies:
+a batch of events contains at least one event but there is no pre-defined upper limit. 
```

Comment:
... a least one event (if you are not already at the end of the journal), but there...

Reply:
done https://github.com/AdobeDocs/adobeio-events/pull/77/commits/5540195f03d94c7adcc96d5c5b629f5fe736ea2f